src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceUtilizationInstruments.cs

```diff
@@ -13,18 +15,42 @@ namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
 internal static class ResourceUtilizationInstruments
 {
     /// <summary>
-    /// Gets the CPU consumption of the running application in range <c>[0, 1]</c>.
+    /// Gets the CPU consumption share of the running process in range <c>[0, 1]</c>.
```

Comment:
I don't believe the summary correctly reflects the nature of the const. It doesn't "_Gets the CPU consumption..._" as it's not a method. It denotes a _**name**_ of an instrument that can provided a value.
With that, I think, the summary should read something like the following:
```suggestion
    /// The name of an instrument to retrieve CPU consumption of the running process in range <c>[0, 1]</c>.
```

@gewarren would you have any suggestions? Thank you.

Reply:
Thanks, good point. I have changed the wording for now